static/app/components/devtoolbar/components/navigation.tsx

```diff
@@ -37,11 +41,11 @@ export default function Navigation({
     >
       <IconButton
         onClick={() => {
-          setIsDisabled(true);
           trackAnalytics?.({
-            eventKey: `devtoolbar.nav.hide.click`,
-            eventName: `devtoolbar: Hide devtoolbar`,
+            eventKey: eventKey + '.hide.click',
```

Comment:
<img width="511" alt="SCR-20240730-mopx" src="https://github.com/user-attachments/assets/29a49609-9e93-471f-834d-05cc5626b21f">

Reply:
added to amplitude